pylzy/lzy/env/python/base.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from abc import abstractmethod
+from typing import NewType, TYPE_CHECKING
+
+from lzy.env.base import Deconstructible
+from lzy.env.explorer.base import ModulePathsList, PackagesDict
+
+if TYPE_CHECKING:
+    from lzy.env.mixin import WithEnvironmentType
+
+PythonVersion = NewType('PythonVersion', str)
```

Comment:
why in this case it is not just a type alias? as in `explorer/base.py` for instance
and do we really have to create type alias just for `str`? you didn't do it for pypi index url for example

Reply:
> why in this case it is not just a type alias? as

well, actually I just tried to use NewType and probably should remove it to be consistent with other code.

> and do we really have to create type alias just for str

This is not type alias, this is new type! 
Read docs about it :)